### ISSUE TITLE
Windows install / Pidgin portable

### ISSUE THREAD

**Issue by GoogleCodeExporter** [marig345/prpltwtr#26]
```
Client (Pidgin, finch, empathy, etc): Pidgin portable
Client Version: 2.1.7
LibPurple Version: 2.7.1
PrplTwtr Version: 0.5.2
Operating System: XP sp3

Problem/Feature request (Be as specific as possible. Steps to reproduce are
especially helpful):
1. The exe installer doesn't detect the installation folder for pidgin portable.
2. The dll plug-in exposed in downloads is outdated.

Could you, please, expose 0.5.2 dll?
```

Original issue reported on code.google.com by `Livius.C...@gmail.com` on 24 Jun 2010 at 5:50

**Comment by GoogleCodeExporter** [comment]
```
Client Version: 2.7.1, not 2.1.7
```

Original comment by `Livius.C...@gmail.com` on 24 Jun 2010 at 5:54

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 0.6.0. 

Autodetection is attempted; if it fails, you can manually specify a path. 
```

Original comment by `mikeage` on 14 Feb 2011 at 6:44
- Changed state: **Verified**
